es-vue-base/src/lib-utils/footer-content.js

```diff
@@ -37,7 +37,7 @@ export default (
             links: [
                 { text: 'Solar calculator', url: `${ES_DOMAIN}/solar/calculator/` },
                 { text: 'Solar rebates', url: `${ES_DOMAIN}/market/equipment-rebates/` },
-                { text: 'News', url: NEWS_DOMAIN, newTab: true },
+                { text: 'News', url: `${ES_DOMAIN}/blog` },
```

Comment:
We should add a trailing slash to `/blog/` here.

Reply:
Fixed! thank you